extension/src/experiments/model/index.ts

```diff
@@ -136,14 +134,15 @@ export class ExperimentsModel extends ModelWithPersistence {
   }
 
   public toggleStatus(id: string) {
+    const experiment = this.getExperimentsAndQueued().find(
+      ({ id: queuedId }) => queuedId === id
+    )
+
     if (
-      isQueued(
-        this.getExperimentsAndQueued().find(
-          ({ id: queuedId }) => queuedId === id
-        )?.status
-      )
+      experiment &&
+      (isQueued(experiment?.status) || isRunningInQueue(experiment))
```

Comment:
While nothing happens, I can still select an experiment running in the queue when I run the `Select Experiments to Display in Plots` command 

<img width="665" alt="image" src="https://user-images.githubusercontent.com/43496356/232787076-6b0e5694-6be1-415d-9873-937f6d182e04.png">